t/lib/t/MusicBrainz/Server/Controller/WS/2/DefaultFormat.pm

```diff
@@ -0,0 +1,91 @@
+package t::MusicBrainz::Server::Controller::WS::2::DefaultFormat;
+
+use JSON qw( decode_json encode_json );
+use Test::Deep qw( cmp_deeply );
+use Test::JSON import => [qw( is_valid_json is_json )];
+use Test::More;
+use Test::Routine;
+use Test::XML::SemanticCompare;
+
+with 't::Mechanize', 't::Context';
+
+test 'MBS-9258: deduction of fmt from Accept header' => sub {
```

Comment:
I'm adding a SearchArtists.pm analogous to BrowseArtists.pm.
However, I think I do see why this is currently not tested (and perhaps might not be testable before mbssss is fully in place): while the other tests can and do use a specifically set up test database, any search request returns data directly from the search server index. So for tests to be reproducible, that would need to be mocked too, using indexes build from the test db.
Will see how it plays out on Jenkins; I may need to comment out the actual tests for later reinstatement.